README.md

```diff
@@ -1,25 +1,58 @@
-* [libplctag.NativeImport](https://www.nuget.org/packages/libplctag.NativeImport/) ![Nuget version](https://img.shields.io/nuget/vpre/libplctag.NativeImport) ![Nuget](https://img.shields.io/nuget/dt/libplctag.NativeImport)
 
-### libplctag
+| Package | Downloads | Stable | Preview |
+|-|-|-|-|
+| [libplctag](https://www.nuget.org/packages/libplctag/) | ![Nuget](https://img.shields.io/nuget/dt/libplctag) | ![Nuget version](https://img.shields.io/nuget/v/libplctag) | ![Nuget version](https://img.shields.io/nuget/vpre/libplctag) |
+| [libplctag.NativeImport](https://www.nuget.org/packages/libplctag.NativeImport/) | ![Nuget](https://img.shields.io/nuget/dt/libplctag.NativeImport) | ![Nuget version](https://img.shields.io/nuget/v/libplctag.NativeImport) | ![Nuget version](https://img.shields.io/nuget/vpre/libplctag.NativeImport) |
+
+
+## libplctag
+
+This is the package intended for use in .NET applications. It depends on libplctag.NativeImport to gain access to the underlying libplctag native library.
 
-This is the package intended for use in .NET applications.
 It provides an API for libplctag that should feel natural to .NET developers by supporting the following features:
 * Values are strongly-typed (both Atomic types and User-Defined Types).
 * Errors are thrown as Exceptions
 * Async/Await
 * Native resource cleanup
 
+
+## libplctag.NativeImport
+
+Most developers will not need to directly reference the Native Import library. This library automatically extracts platform-specific version of the base libplctag library needed for the libplctag .NET wrapper. 
+
+If you wish to override this behavior you can do so: [Using a non packaged version of the native libplctag library](https://github.com/libplctag/libplctag.NET/wiki/Using-a-non-packaged-version-of-the-native-libplctag-library)
+
+
+Documentation for the base library API can be found [here](https://github.com/libplctag/libplctag/wiki/API). Further examples of its usage can be found [here](https://github.com/libplctag/libplctag.NET/blob/master/src/Examples/CSharp%20DotNetCore/NativeImportExample.cs).
+
+The libplctag native library can be compiled for [many platforms](https://github.com/libplctag/libplctag#platform-support), and not all supported platforms are shipped with this wrapper. If you get a `TypeLoadException`, chances are that you can still use this wrapper but you will need to [supply the runtime yourself](https://github.com/libplctag/libplctag/blob/master/BUILD.md).
+
+
+## Getting Started
+
+In most cases only the  libplctag package will be needed. It can be added in Visual Studio through the package manager or via the commandline:
+
+`dotnet add package libplctag`
+
+### Example Code for an Allen-Bradley CompactLogix/ControlLogix PLC
+
 ```csharp
 // Instantiate the tag with the appropriate mapper and datatype
 var myTag = new Tag<DintPlcMapper, int>()
 {
+    //Name is the full path to tag. "PROGRAM:" is required for C*Logix.
```

Comment:
Global tags don't require "PROGRAM:"
I believe the gateway IP addess could refer to an ENBT, as opposed to the Controller. When I hear the term PLC I assume the Controller.

I think we should avoid putting advisory comments like this in and should defer to libplctag core for usage and explanation.
It would be good if there was something in libplctag core that talked about these - there is a lot of variety.

Reply:
Dang it. I was trying to give people a good starting point instead of "look up documentation and examples in all these different places". Not that there's anything wrong with that once people need to do more than the simplest hello world, but I don't think our first code example should require that much understanding of everything.

I want to split the difference between advisory comments, and people just copying things that they have no understanding of.